src/sessions/session-manager.ts

```diff
@@ -121,6 +121,9 @@ export class SessionManager {
   }
 
   async startSession (startSessionParams: IStartSessionParams | IStartVideoSessionParams | IStartSoftphoneSessionParams): Promise<any> {
+    if(!this.sdk.connected) {
+      throw createAndEmitSdkError.call(this.sdk, SdkErrorTypes.session, 'A session cannot be started as streaming client is not yet connected', {startSessionParams});
```

Comment:
we can't log the sessionStartParams because it could have PII in it such as a phone number or user jid. Please look through the start params and see what we can log.